update/delta/delta_1.3.6_add_vl_for_SIA_export.sql

```diff
@@ -0,0 +1,237 @@
+UPDATE qwat_vl.pipe_material SET sia405code = 'Kunststoff.Glasfaserverstaerkter_Epoxiharz' WHERE short_fr = 'GRP';
+UPDATE qwat_vl.pipe_material SET sia405code = 'Kunststoff' WHERE short_fr = 'PL';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.pipe_protection SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : Planimetric precision mapping to complete [X] to check [ ]
+UPDATE qwat_vl.precision SET sia405code = 'unbekannt' WHERE value_fr = 'Autre';
+UPDATE qwat_vl.precision SET sia405code = 'unbekannt' WHERE value_fr = 'Inconnu';
+UPDATE qwat_vl.precision SET sia405code = 'unbekannt' WHERE value_fr = 'À déterminer';
+UPDATE qwat_vl.precision SET sia405code = 'ungenau' WHERE value_fr = 'Imprécis';
+UPDATE qwat_vl.precision SET sia405code = 'genau' WHERE value_fr = 'Précis';
+UPDATE qwat_vl.precision SET sia405code = 'unbekannt' WHERE value_fr = 'Localisé';
+UPDATE qwat_vl.precision SET sia405code = 'unbekannt' WHERE value_fr = 'Digitalisé';
+
+-- TODO : Altimetric precision mapping to complete [X] to check [ ]
+UPDATE qwat_vl.precisionalti SET sia405code = 'unbekannt' WHERE value_fr = 'Autre';
+UPDATE qwat_vl.precisionalti SET sia405code = 'unbekannt' WHERE value_fr = 'Inconnu';
+UPDATE qwat_vl.precisionalti SET sia405code = 'unbekannt' WHERE value_fr = 'À déterminer';
+UPDATE qwat_vl.precisionalti SET sia405code = 'ungenau' WHERE value_fr = 'Imprécis';
+UPDATE qwat_vl.precisionalti SET sia405code = 'ungenau' WHERE value_fr = 'Inférieure à 100 cm';
+UPDATE qwat_vl.precisionalti SET sia405code = 'ungenau' WHERE value_fr = 'Inférieure à 50 cm';
+UPDATE qwat_vl.precisionalti SET sia405code = 'genau' WHERE value_fr = 'Inférieure à 10 cm';
+UPDATE qwat_vl.precisionalti SET sia405code = 'genau' WHERE value_fr = 'Inférieure à 5 cm';
+
+-- TODO : complete mapping
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.unbekannt' WHERE value_fr = 'à déterminer';
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.Druckbrecher' WHERE value_en = 'reducer';
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.Druckbrecher' WHERE value_en = 'pressure cut';
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.unbekannt' WHERE value_en = 'gathering';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.protectionzone_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.pump_operating SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.pump_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.remote_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.source_quality SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
```

Comment:
Wassergewinnungsanlage.Art
Looks complete to me

Reply:
TOp!